builder.py

```diff
@@ -0,0 +1,208 @@
+# Taken from SELU implementation from https://github.com/bioinf-jku/SNNs/blob/master/selu.py
+import os
+
+import tensorflow as tf
+import random
+'''
+Tensorflow Implementation of the Scaled ELU function and Dropout
+Taken from https://github.com/hunkim/DeepLearningZeroToAll
+'''
+import numbers
+from tensorflow.contrib import layers
+from tensorflow.python.framework import ops, tensor_shape, tensor_util
+from tensorflow.python.ops import math_ops, random_ops, array_ops
+from tensorflow.python.layers import utils
+
+from constants import *
+
```

Comment:
The format of import statements should always be

1) 3rd party / python built-in imports in alphabetical order. don't differentiate between "from .... import x" and "import y". add an empty line at the end of all such imports

2) local imports.